pkgs/servers/seafile/seafile-server.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, lib, fetchFromGitHub, pkg-config, python3Packages, autoreconfHook
+, libuuid, sqlite, glib, libevent, libsearpc, openssl, fuse, libarchive, which
+, vala, cmake }:
+
+let
+  libevhtp = import ./libevhtp.nix {
+    inherit stdenv lib fetchFromGitHub cmake libevent;
+  };
+  oniguruma =
+    import ./oniguruma.nix { inherit stdenv lib fetchFromGitHub cmake; };
```

Comment:
```suggestion
  oniguruma = import ./oniguruma.nix { inherit stdenv lib fetchFromGitHub cmake; };
```

Reply:
I removed the extra newline but those files were formatted with nixfmt, am I missing some configuration option to achieve this?